pkg/deploy/kapp.go

```diff
@@ -231,3 +270,20 @@ func (a *Kapp) addGenericArgs(args []string, appName string) ([]string, []string
 
 	return args, env
 }
+
+// trySaveAppMeta if unable to save the kapp configmap metadata, then continue and do not fail the deploy.
+func (a *Kapp) trySaveAppMeta(metadataFileName string) {
+	metadataFile, err := os.ReadFile(metadataFileName)
+	if err != nil {
+		a.appMeta = nil
+		return
+	}
+
+	appMetadata := Meta{}
+	err = yaml.Unmarshal(metadataFile, &appMetadata)
+	if err != nil {
```

Comment:
reset a.appMeta = nil here as well? or should we just do it at the top of the Deploy() function so that we dont have to think about all the error cases.

Reply:
Nice! added it to the top of Deploy()